### ISSUE TITLE
How to add twitter:card to the meta using vue-meta

### ISSUE THREAD

**Issue by moda20** [nuxt/vue-meta#817]
i am trying to add a simple twitter card and title as meta tags to my vue3 pages, the tags are different between the pages,
i can't seem to find any documentation that can show exactly how to do that, everything is super complicated. 

i tried adding `meta` attribute with the desired attrobutes to the vue-router routes and return it in `metaInfo` function but that didn't work.

**Comment by amshiro** [comment]
Hi, bro, have you solved this problem?

**Comment by moda20** [comment]
@amshiro no, i dropped the package and added the metatags to the index.html directly. there is head.js lib that is a better alternative for this one

**Comment by amshiro** [comment]
> @amshiro no, i dropped the package and added the metatags to the index.html directly. there is head.js lib that is a better alternative for this one

I haven't implemented it either. I'll try head.js. Thanks for your suggestion.

**Comment by amshiro** [comment]
> > @amshiro no, i dropped the package and added the metatags to the index.html directly. there is head.js lib that is a better alternative for this one
> 
> I haven't implemented it either. I'll try head.js. Thanks for your suggestion.

Bro, would you like to share the core usage of head.js? 😊🙏😊

**Comment by mateusfmello** [comment]
Wouldn't `<teleport to="head">` be an alternative to this problem?
